scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/defaults/main.yml

```diff
@@ -33,6 +33,7 @@ ardana_qe_tests_requires_state: "present"
 ardana_qe_tests_requires:
   - "gcc"
   - "python-devel"
+  - "java-1_8_0-openjdk-headless"
```

Comment:
As `java-1_8_0-openjdk-headless` is not required for other tests, can you overwrite the `ardana_qe_tests_requires` variable just for keystone tests?